pyproject.toml

```diff
@@ -22,7 +22,7 @@ target-version = ["py310"]
 profile = "black"
 
 [tool.poetry]
-# name = "opensearch-dashboards-operator"
```

Comment:
Kept `tool.pytest.ini_options` without `async_mode=auto`. We had an issue on opensearch because of that.